tests/decorators/test_cosmosdb.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import unittest
+
+from azure.functions import DataType
+from azure.functions.decorators.core import BindingDirection
+from azure.functions.decorators.cosmosdb import CosmosDBTrigger, \
+    CosmosDBOutput, CosmosDBInput
+
+
+class TestCosmosDB(unittest.TestCase):
+    def test_cosmos_db_trigger_valid_creation(self):
```

Comment:
Also add a test scenario where you are sending only the minimal stuff - we want to check the function.json when Nones are automatically set

Reply:
This comment applies to all the triggers and bindings.

Reply:
Need not be part of this PR though

Reply:
Okay - I got my partial answer - you are validating function.json with fewer args.